src/gitProviders/github.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from "react";
+import axios from 'axios';
+
+export class GitHub extends React.Component<any, any>{
+
+    constructor(props) {
+        super(props);
+        console.log(this.context);
```

Comment:
See other comment about logging.
Maybe use `console.debug()` to restrict logging to verbose out and keep default stdout clean and add context beforehand :)

```
console.debug("GitHub component context:", this.context);
```